### PR TITLE
:hammer: Changed NDV opening behavior on node duplication N8N-2896

### PR DESCRIPTION
**Pull request by michael-radency**

### CODE REVIEW

File: packages/editor-ui/src/views/NodeView.vue [hunk 1]
```diff
@@ -1979,9 +1979,9 @@ export default mixins(
 				// Automatically deselect all nodes and select the current one and also active
 				// current node
 				this.deselectAllNodes();
-				setTimeout(() => {
-					this.nodeSelectedByName(newNodeData.name, true);
-				});
+				// setTimeout(() => {
```
Comment (mutdmour) [inline packages/editor-ui/src/views/NodeView.vue:1982]: would be better if you simply delete this

File: packages/editor-ui/src/views/NodeView.vue [hunk 2]
```diff
@@ -1979,9 +1979,6 @@ export default mixins(
 				// Automatically deselect all nodes and select the current one and also active
 				// current node
 				this.deselectAllNodes();
-				setTimeout(() => {
-					this.nodeSelectedByName(newNodeData.name, true);
-				});
```
Comment (mutdmour) [inline packages/editor-ui/src/views/NodeView.vue:1984]: this does more than just not open NDV. this also selects the node. we should keep the behaviour that new node is selected.

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.162.0`

**Pull request merged**